src/index.html

```diff
@@ -11,7 +11,139 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
+      <fieldset class="fieldset">
+        
+        <legend>Personal information</legend>
+
+        <label class="form_field">
+          Surname:
+          <input
+            name="surname"
+            type="text"
+            autocomplete="off"
+            required
+          ></input>
+        </label>
+        <label class="form_field">
+          Name:
+          <input
+            name="name"
+            type="text"
+            autocomplete="off"
+            required
+          ></input>
+        </label>
+        <label class="form_field">
+          How old are you?
+          <input
+            name="age"
+            type="number"
+            min="1"
+            max="100"
+            value="12"
+          ></input>
+        </label>
+        <label class="form_field">
+          Full date of birth:
+          <input
+            name="date"
+            type="date"
+            required
+          ></input>
+        </label>
+        <label class="form_field">
+          I accept the term of the agreement
+          <input
+            name="term"
+            type="checkbox"
+            required
+          ></input>
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+
+        <legend>Registration</legend>
+
+        <label class="form_field">
+          E-mail:
+          <input
+            name="email"
+            type="email"
+            placeholder="email@example.com"
+          ></input>
+        </label>
+        <label class="form_field">
+          Password:
+          <input name="password" type="password"></input>
+        </label>
+
+      </fieldset>
+      <fieldset class="fieldset">
+
+        <legend>An interesting fact about you</legend>
+
+        <div class="form_field">
+          Do you love cats?
+          <label>
+            <input name="loveCats" type="radio"></input>
```

Comment:
You don't need to close tag `<input>`
```suggestion
            <input name="loveCats" type="radio">
```